### ISSUE TITLE
Spurious identifier not found error

### ISSUE THREAD

**Issue by disarray2077** [beefytech/Beef#1659]
The below code compiles fine but sometimes the IDE complains about `sso` not being found:
![image](https://user-images.githubusercontent.com/86157825/180658029-7eb3e485-ba15-4e1d-8cc8-a7da8bfaaaa2.png)

Code:
```cs
	void Test1()
	{
		StringSplitOptions sso = .None;

		void Test2()
		{
			switch (sso)
			{
			case .None:
				break;
			case .RemoveEmptyEntries:
				break;
			}
		}

		Test2();
	}

	Test1();
```

Tested with: https://github.com/beefytech/Beef/commit/119e9189e5cc41c747b2af01abbb87966cb6b422

**Comment by bfiete** [comment]
I'm having some trouble reproducing this... I wonder if I'm missing some important element to the repro...

**Comment by disarray2077** [comment]
I tested with it for a bit and this is what I noticed:
- If you hover over `sso` (in `switch (sso)`), `.None`, `.RemoveEmptyEntries` the error goes away.
- If you hover over anything else that shows a tooltip, the error comes back.

**Comment by bfiete** [comment]
Fixed at e247e3ecb4ae46f96ae4d3de6cceac73b80920b3
